show-land/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WeProDev BeTheme</title>
+
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+
+   <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+
+    <div class="background-color">
+
+        <div class="container">
+            <header class="header ">
+                <h1 class="logo">Anvogue</h1>
+                <nav class="main-nav">
+                    <ul class="main-nav__list">
+                        <li> <a class="main-nav__link" href="#">demos</a> </li>
+            <li><a class="main-nav__link" href="#">mobile view</a> </li>
+            <li><a class="main-nav__link" href="#">features</a> </li>
+            <li><a class="main-nav__link" href="#">help & support</a> </li>
+            <li><a class="main-nav__link btn btn-buttom nav-cta" href="#">purchase now</a> </li>
+        </ul>
+    </nav>
+</header>          
+</header>
+<section class="hero">hero</section>
+</div>
+    </div>
+<main>
+
+    <section class="super-fast-section">super fast</section>
+    <section class="Home-page-section">Home page </section>
+    <section class="page-builder-section">page builder section</section>
+    <section class="benefits-section">Benefits section </section>
+    <section class="features-section ">features section </section>
+    <section class="powerful-features-section">Powerful features section</section>
+    <section class="help-users-section">Help users section</section>
+    <section class="trusted-section ">trusted section </section>
+    <section class="footer-section">footer section</section>
+</div>
+
+</main>
+    
+</div>
+</div>
+
+    </body>
+    </html>
+   
```

Comment:
what is this section for?

Reply:
that's it for the next section

Reply:
If you want to write the next section you have to create a new branch for it

For example you create a branch for Header you have to write the headers code nothing more

Reply:
If you want to write the next section you have to create a new branch for it

For example you create a branch for Header you have to write the headers code nothing more

Reply:
It is better for each branch to be specific to each section

Much cleaner, **what do you think?**